environment-templates/fargate-env/v1/infrastructure/main.tf

```diff
@@ -0,0 +1,54 @@
+module "vpc" {
+  source = "terraform-aws-modules/vpc/aws"
+
+  cidr = var.environment.inputs.vpc_cidr
+
+  azs                  = [data.aws_availability_zones.available.names[0], data.aws_availability_zones.available.names[1]]
+  private_subnets      = [var.environment.inputs.private_subnet_one_cidr, var.environment.inputs.private_subnet_two_cidr]
+  public_subnets       = [var.environment.inputs.public_subnet_one_cidr, var.environment.inputs.public_subnet_two_cidr]
+  enable_nat_gateway   = true
+  enable_vpn_gateway   = true
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+
+  tags = {
+    Terraform   = "true"
+    Environment = var.environment.name
+  }
+}
+
+resource "aws_service_discovery_private_dns_namespace" "service_discovery" {
+  name        = "${var.environment.name}.local"
+  description = ""
+  vpc         = module.vpc.vpc_id
+}
+
+resource "aws_sns_topic" "ping_topic" {
+  name              = "${var.environment.name}-ping"
+  kms_master_key_id = "alias/aws/sns"
+}
+
+resource "aws_ecs_cluster" "fargate_cluster" {
+  name = "${var.environment.name}-Cluster"
+  capacity_providers = [
```

Comment:
Documentation says these fields are deprecated in favor of `aws_ecs_cluster_capacity_providers`

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_cluster